src/note/note.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable, Logger } from '@nestjs/common';
+import {InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { notes } from './note.entity';
+import { Note } from '../interfaces/note'
+@Injectable()
+export class  NoteService {
+constructor(
+@InjectRepository(notes)
+private noteRepository: Repository<notes>,
+) {}
+async index(): Promise<Note[]> {
+return await this.noteRepository.find();
+}
+async findOne(id): Promise<Note[]>{
+return await this.noteRepository.find({where: [{ 'id': id }]});
+}
+async create(note: Note): Promise<any> {
+ const noteReady = await this.noteRepository.save(note);
```

Comment:
tutaj powinno być
return this.noteRepository.save(note);
żeby funkcja, która wywołuje potem noteService.create() widziała efekt i wtedy będzie się dało zmienić typ całej metody
z Promise<any> na Promise<Note>